sim/midas/src/main/scala/midas/core/Channel.scala

```diff
@@ -241,6 +241,8 @@ class ReadyValidChannelUnitTest(
 
   val dut = Module(new ReadyValidChannel(payloadType))
   val reference = Module(new ShiftQueue(payloadType, queueDepth))
+  val referenceReset = Wire(Bool())
+  reference.reset := referenceReset
```

Comment:
Why does this need to be a wire?

Reply:
reference.reset is an abstract Reset. For the LIBDN tester hardware to generate correctly it needs to be a concrete reset -- specifically a Bool in this case.  Using the wire here both gives us a reference to drive from the LIBDN tester the and forces a particular concrete type on to the port.